app/controllers/application_controller.rb

```diff
@@ -9,26 +9,34 @@ def current_intake
     current_user&.intake || Intake.find_by_id(session[:intake_id])
   end
 
+  def current_diy_intake
+    DiyIntake.find_by_id(session[:diy_intake_id])
+  end
+
   def include_google_analytics?
     false
   end
 
+  def visitor_record
+    current_intake
+  end
```

Comment:
This is an interesting abstraction. It might be something that we could expand beyond just our handling of the visitor_id. A generic "key model" or "navigation record" or something could be used to make all our flow, forms, and other things more modular and less dependent on a specific type of model. For example, what if the personal info form could work on any model that had preferred name and state of residence fields?